Makefile

```diff
@@ -40,7 +40,8 @@ ifeq ($(REGISTRY_ENABLED),true)
 ifeq ($(TOOL),oc)
 	$(TOOL) apply -f ./deploy/registry/local/os
 else
-	sed -i "s|192.168.99.100|$(CLUSTER_IP)|g" ./deploy/registry/local/k8s/ingress.yaml
+	sed -i.bak -e "s|192.168.99.100|$(CLUSTER_IP)|g" ./deploy/registry/local/k8s/ingress.yaml
+	rm ./deploy/registry/local/k8s/ingress.yaml.bak
```

Comment:
Could you also mirror this commit from my PR: https://github.com/che-incubator/che-workspace-operator/pull/50/commits/83cb27d073d6d47cf474107e19aef2b6652bf75f? We're currently not resetting `k8s/ingress.yaml` after creating the route.